packages/desktop/src/main/main.ts

```diff
@@ -32,16 +32,11 @@ const webcrypto = new Crypto()
 
 global.crypto = webcrypto
 
-let dataDir = DESKTOP_DATA_DIR
 let mainWindow: BrowserWindow | null
 let splash: BrowserWindow | null
 let invitationUrl: string | null
 
-if (isDev || process.env.DATA_DIR) {
```

Comment:
This logic just got moved to `loadMainEnvs` so we could ensure the log env variables are set as early as possible.